swaymsg/main.c

```diff
@@ -389,7 +389,7 @@ int main(int argc, char **argv) {
 		type = IPC_COMMAND;
 	} else if (strcasecmp(cmdtype, "get_workspaces") == 0) {
 		type = IPC_GET_WORKSPACES;
-	} else if (strcasecmp(cmdtype, "get_seats") == 0) {
+        } else if (strcasecmp(cmdtype, "get_seats") == 0) {
```

Comment:
The indentation here is wrong, but there is also no need to alter this file

Reply:
Removed from commit as well.